src/pages/Detail/component/DetailIntro.jsx

```diff
@@ -0,0 +1,360 @@
+      .then(response => response.json())
+      .then(result => setReviews(result.review));
+  }, [offset]);
+
+  const maxPageNum = Math.ceil(totalReview?.length / 4);
+  const currentPage = offset / 4 + 1;
+
+  const pageArr = (totalReview => {
+    let result = [];
+    for (let i = 1; i <= maxPageNum; i++) {
+      result.push(i);
+    }
+    return result;
+  })(totalReview);
+
+  const movePage = pageNum => {
+    setOffset((pageNum - 1) * 4);
+  };
+
+  const prevBtn = () => {
+    if (currentPage === 1) return;
+    setOffset(0);
+  };
+
+  const nextBtn = () => {
+    if (currentPage === maxPageNum) return;
+    setOffset(currentPage * 4);
+  };
+
+  // API주소/review/classId=${classId}&reviewId=${Number(e.target.id)}
+
+  const deleteBtn = e => {
+    e.preventDefault();
+    fetch(
+      `API주소/review/classId=${classId}&reviewId=${Number(e.target.id)}`,
+      {
+        method: 'DELETE',
+        header: {
+          // authorization: token,
+          'Content-Type': 'application/json;charset=utf-8',
+        },
+      }
+        .then(response => response.json())
+        .then(result => setReviews(result.review))
+    );
+  };
+
+  if (!reviews) return <>loading...</>;
+
+  return (
+    <>
+      <InfoTitle>클래스 정보</InfoTitle>
+      <InfoText1>클래스 분량</InfoText1>
+      <InfoText2>5개 챕터, 30개 세부강의</InfoText2>
+      <InfoText1>수강 가능일</InfoText1>
+      <InfoText2>바로 수강 가능</InfoText2>
+      <InfoText1>자막 포함 여부</InfoText1>
+      <InfoText2>포함</InfoText2>
+      <Line id="reviewConer" />
+
+      <ReviewTitle>
+        <Title>실제 수강생 후기</Title>
+        <ReviewBtn onClick={modalHandler}>후기 작성하기</ReviewBtn>
+      </ReviewTitle>
+
+      {isModalOpen && (
+        <Modal
+          isOpen={isModalOpen}
+          onRequestClose={modalHandler}
+          style={modalDisplay}
+        >
+          <XBtn onClick={modalHandler}>
+            <AiOutlineClose />
+          </XBtn>
+          <ReviewModal
+            classId={classId}
+            classTitle={classTitle}
+            closeModal={modalHandler}
+            selectModal={selectModal}
+            setReviews={setReviews}
+          />
+        </Modal>
+      )}
+
+      {isEdit && (
+        <Modal
+          isOpen={isEdit}
+          onRequestClose={editModalHandler}
+          style={modalDisplay}
+        >
+          <XBtn onClick={editModalHandler}>
+            <AiOutlineClose />
+          </XBtn>
+          <ReviewModal
+            classId={classId}
+            classTitle={classTitle}
+            closeModal={isEdit}
+            isModalOpen={isEdit}
+            selectModal={selectModal}
+            setReviews={setReviews}
+            registTxt={registTxt}
+          />
+        </Modal>
+      )}
+
+      <ImageWrap>
+        {reviews?.map(item => {
+          return item.image_url === '' ? null : (
+            <ImageContainer key={item.review_id}>
+              <ReviewImg src={item.image_url} alt="후기 이미지" />
+            </ImageContainer>
+          );
```

Comment:
```suggestion
          return item.image_url !== && (
            <ImageContainer key={item.review_id}>
              <ReviewImg src={item.image_url} alt="후기 이미지" />
            </ImageContainer>
          );
```

삼항 연산자 값이 하나가 null이라면 && 연산자를 활용하는 것이 더 좋겠네요!